samples/geography/requirements.txt

```diff
@@ -10,13 +10,13 @@ dataclasses==0.6; python_version < '3.7'
 Fiona==1.8.20
 geojson==2.5.0
 geopandas==0.9.0
-google-api-core==1.31.2
-google-auth==1.35.0
+google-api-core==2.0.0
+google-auth==2.0.1
```

Comment:
Bumping the `google-api-core` / `google-auth` pins to `2.x` for samples should be OK:  we aren't supposed to update them in `setup.py` until after https://github.com/googleapis/google-cloud-python/issues/10566 is closed.